src/textediteventfilter.cpp

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: 2023 Mikhail Zolotukhin <mail@gikari.com>
```

Comment:
For the sake of correctness I think the copyright header for the newly created files that are mostly based on the old code should be probably like this:
```suggestion
// SPDX-FileCopyrightText: 2023 Mikhail Zolotukhin <mail@gikari.com>
// SPDX-FileCopyrightText: 2018-2019 Sassan Haradji <sassanh@gmail.com>
```